staging/src/k8s.io/pod-security-admission/admission/api/validation/validation.go

```diff
@@ -18,11 +18,70 @@ package validation
 
 import (
 	"k8s.io/apimachinery/pkg/util/validation/field"
-	"k8s.io/pod-security-admission/admission/api"
+	admissionapi "k8s.io/pod-security-admission/admission/api"
+	"k8s.io/pod-security-admission/api"
 )
 
-func ValidatePodSecurityConfiguration(configuration *api.PodSecurityConfiguration) field.ErrorList {
+// ValidatePodSecurityConfiguration validates a given PodSecurityConfiguration.
+func ValidatePodSecurityConfiguration(configuration *admissionapi.PodSecurityConfiguration) field.ErrorList {
```

Comment:
range over Exemptions.Namespaces and ensure they pass `k8s.io/apimachinery/pkg/api/validation#ValidateNamespaceName` and contain no duplicates

range over Exemptions.RuntimeClasses and ensure they pass `k8s.io/apimachinery/pkg/api/validation#NameIsDNSSubdomain` and contain no duplicates

range over Exemptions.Usernames and ensure they are not empty and contain no duplicates